app/cafe.py

```diff
@@ -0,0 +1,18 @@
+import datetime
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("Vaccine is not")
+        elif visitor["vaccine"]["expiration_date"] < datetime.date.today():
+            raise OutdatedVaccineError("Vaccine is outdated")
+        elif not visitor.get("wearing_a_mask", False):
+            raise NotWearingMaskError("Wearing mask is not")
```

Comment:
The error message for `NotWearingMaskError` is not descriptive enough. A more descriptive message could be 'Visitor is not wearing a mask.'